flink-runtime/src/main/java/org/apache/flink/runtime/state/memory/MemoryStateBackend.java

```diff
@@ -331,13 +331,18 @@ public OperatorStateBackend createOperatorStateBackend(
 		TaskStateManager taskStateManager = env.getTaskStateManager();
 		HeapPriorityQueueSetFactory priorityQueueSetFactory =
 			new HeapPriorityQueueSetFactory(keyGroupRange, numberOfKeyGroups, 128);
+
+		ReadableConfig config = env.getTaskManagerInfo().getConfiguration();
+		int fsReadBufferSize = config.get(CheckpointingOptions.FS_READ_BUFFER_SIZE);
```

Comment:
Same as above, please do this inside of `configure`.

Reply:
done